packages/imperative/src/config/cmd/convert-profiles/convert-profiles.handler.ts

```diff
@@ -11,6 +11,8 @@
 
```

Comment:
Do we want to disable the following check if `--delete` flag is true?
https://github.com/zowe/imperative/blob/9261c337222e2c9dcc40222988c1228f1975a62d/packages/imperative/src/config/cmd/convert-profiles/convert-profiles.handler.ts#L63-L66

Currently, it appears that if someone has already run `zowe config convert-profiles`, then they can't run `zowe config convert-profiles --delete` afterwards, since they have 0 active v1 profiles and this code block will exit the handler.

Reply:
I might just add an if statement around the return.